priv/repo/migrations/20170515154812_rename_user_github_id_to_github_auth_token.exs

```diff
@@ -0,0 +1,7 @@
+defmodule CodeCorps.Repo.Migrations.RenameUserGithubIdToGithubAuthToken do
+  use Ecto.Migration
+
+  def change do
+    rename table(:users), :github_id, to: :github_auth_token
```

Comment:
I did not consider that. We'd need it as part of the integration feature. Technically, however, all the connect feature needs is the `auth_token`.